### ISSUE TITLE
Button group - dropdown items fail to receive focus

### ISSUE THREAD

**Issue by ghost** [vmware/clarity#5256]
## Describe the accessibility issue

I am using my keyboard to interact with a clr-button-group. The button group has several elements in a dropdown ([clrInMenu]="true").
When I focus the dropdown and open it there is no way to interact with the buttons in the list by only using the keyboard.
Instead the focus moves to the next focusable element on the page.

The button group works well when there is nothing else on the page. In my case however there are several items on the DOM which have their tabindex attribute set to 0. They receive focus instead of the buttons in the group.

To summarize - there is no way to use the clr-button-group dropdown with the keyboard if there is an element anywhere on the page which has tabindex="0"

## How to reproduce

https://stackblitz.com/edit/clarity-v3-light-theme-2ae6ca
Use keyboard to open the dropdown and try to move focus using TAB.

## Expected behavior

It should be possible to focus the buttons in the dropdown. Ideally when the dropdown opens the focus
should immediately move to the first item in the list.

## Versions

**App**

- Clarity: 3.1.3

**Device:**

- Type: Macbook
- Browser: Chrome

**Comment by Jinnie** [comment]
Seems to work fine in v2, but not in the later versions.
